Lib/asyncio/selector_events.py

```diff
@@ -618,19 +618,24 @@ def _sock_sendto(self, fut, sock, data, address):
         else:
             fut.set_result(n)
 
-    async def sock_connect(self, sock, address):
+    async def sock_connect(self, sock, address, proto_addr_info = None):
         """Connect to a remote socket at address.
 
         This method is a coroutine.
+        
+        The proto_addr info parameter allows to specify a protocol to use for the 
+        resolution of the address. If not specified, default to the protocol used
+        by sock
```

Comment:
I don't think this is needed :(
```suggestion
```
We don't have a docstring for any of the other parameters.